### PR TITLE
Adding meta/extensions.yml for displaying EDA content on Automation Hub

### PR DESCRIPTION
**Pull request by alisonlhart**
The method of displaying EDA content on Automation Hub is through the `meta/extensions.yml` file. This is planned to be used eventually by ansible-doc to display EDA content, and currently is used directly by galaxy-importer to display EDA content.

Currently the ansible.eda 's EDA plugins do not show up on Automation Hub. With the addition of this file, they display under the collection's listed "content".

### CODE REVIEW

**Review by eclarizio — APPROVED** [review]
:+1:

**Review by Alex-Izquierdo — APPROVED** [review]
LGTM, failed CI should pass after merge https://github.com/ansible/event-driven-ansible/pull/167

**Pull request merged**
